### PR TITLE
Resolve VCSWP-22704

### PR DESCRIPTION
**Pull request by amahadaya**
Fix test files and SDK file auto-generation

### CODE REVIEW

File: spec/models/account_result_spec.rb [hunk 1]
```diff
@@ -221,41 +217,14 @@
...
-          account_id: "ST",
-          api_key: "ST",
-          _alias: "ST",
-          label: "ST",
-          type: "FULL",
-          status: "SUSPENDED",
-          subresource_uris: nil,
-      )
-      expect(instance_1.eql?(instance_2)).to eq(false)
-    end
```
Comment (ngarg-vail) [inline spec/models/account_result_spec.rb:258]: Why was this removed?

File: spec/models/account_result_spec.rb [hunk 2]
```diff
@@ -416,7 +385,7 @@
       
         date_updated: "TS",
       
-        revision: 1,
+revision: 1,
```
Comment (ngarg-vail) [inline spec/models/account_result_spec.rb]: Formatting is off in this file

Comment (amahadaya) [reply spec/models/account_result_spec.rb]: Not according to this file link: https://github.com/FreeClimbAPI/ruby-sdk/blob/6d3a3f8a6dd5d1266cf56795ea0fb3dfe478aceb/spec/models/account_result_spec.rb

Comment (ngarg-vail) [reply spec/models/account_result_spec.rb]: See screenshot
image

File: spec/models/add_to_conference_spec.rb [hunk 3]
```diff
@@ -148,16 +148,16 @@
       )
       expect(instance.valid?).to eq(true)
     end
+
     it 'checks if properties are invalid' do
       instance = Freeclimb::AddToConference.new(
-          conference_id: nil,
+      allow_call_control: nil,                  
+      
+      conference_id: nil,            listen: nil,            start_conf_on_enter: nil,      talk: nil,
```
Comment (ngarg-vail) [inline spec/models/add_to_conference_spec.rb]: this looks wrong

File: spec/models/application_list_spec.rb [hunk 4]
```diff
@@ -28,43 +28,61 @@
   
   describe 'test attribute "total"' do
     it 'should work' do
-      instance.total = 1
-      expect(instance.total).to eq(1) 
+    
+    
+    instance.total = 1
+    expect(instance.total).to eq(1)
```
Comment (ngarg-vail) [inline spec/models/application_list_spec.rb]: formatting  + unnecessary new lines

File: spec/models/add_to_conference_spec.rb [hunk 5]
```diff
@@ -213,7 +214,7 @@
       expect(instance_1.eql?(instance_2)).to eq(false)
     end
   end
-
+
```
Comment (ngarg-vail) [inline spec/models/add_to_conference_spec.rb:230]: extra spaces

**Review by ngarg-vail — CHANGES_REQUESTED** [review]

**Review by ngarg-vail — APPROVED** [review]

**Review by ngarg-vail — APPROVED** [review]
